src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, currentTab, onTabSelected }) => {
+  const onClickTabHandler = (tab) => {
+    if (tab !== currentTab) {
+      onTabSelected(tab);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                className={currentTab.id === id ? 'is-active' : ''}
```

Comment:
Use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally